### PR TITLE
Mc/feature/build modal template

### PR DESCRIPTION
**Pull request by Maia-Cochran**
# Pull Request

- [x] New feature (adds functionality)
- [x] Describe the purpose of this PR (other): adds modal template for reusable modal (i.e. for navbar and the carousel images)
- [x] I have added screenshots, when necessary (see below):

# Please describe the tests that you ran to verify your changes.

- [x] I tested my changes in the simulator
- [ ] I tested with Cypress

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new bugs
- current rendering of ModalTemplate button pushes positioning of navbar 
- this will be solved once connected, leaving as is for view by team
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any other comments, questions or concerns?
- How can we connect the modal to the carousel and navbar?

Screen Shot 2022-09-13 at 8 20 49 PM

Screen Shot 2022-09-13 at 8 21 33 PM

### CODE REVIEW

File: components/ModalTemplate.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { Button, Text, View, Modal, StyleSheet, Pressable } from "react-native";
+
```
Comment (GOECHA) [inline components/ModalTemplate.js:3]: Nice use of the hook useState !!!

**Review by GOECHA — APPROVED** [review]
Great job getting the modal initialized!

**Comment by GOECHA** [comment]
Great job getting the modal started!!!

**Comment by DrSLMac** [comment]
Awesome job!

**Pull request merged**